roc_example_test.go

```diff
@@ -77,7 +79,8 @@ func ExampleROC_equallySpacedCutoffs() {
 	n := 9
 
 	stat.SortWeightedLabeled(y, classes, weights)
-	cutoffs := stat.EquallySpaced(y[0], y[len(y)-1], n)
+	cutoffs := make([]float64, n)
+	floats.Span(cutoffs, math.Nextafter(y[0], y[0]-1), y[len(y)-1])
```

Comment:
>In principle, if you wanted it to be symmetric, you should span from just below the minimum value to just above the maximum value.

That breaks a different symmetry. Are these not always handled as a set of disjoint intervals [c_n, c_{n+1})? This is how `stat.Histogram` does it. The logic I gave above gives this without depending on the user to provide a carefully prepared set of intervals. (As an aside, now that we are considering this new approach of providing cutoffs, there are reasons why this might be good to provide for `stat.Histogram` or a relative).